mzLib/MassSpectrometry/PeakIndexing/ExtractedIonChromatogram.cs

```diff
@@ -16,10 +16,8 @@ namespace MassSpectrometry
     public class ExtractedIonChromatogram
     {
         public virtual List<IIndexedPeak> Peaks { get; set; }
-
-        public double ApexRT;
-        public int ApexScanIndex;
```

Comment:
You can leave these properties, just make them expression bodied members that point to the respective fields in ApexPeak, e.g.,
`ApexRT => ApexPeak.RetentionTime`